synapse/storage/databases/main/room.py

```diff
@@ -1998,6 +1998,9 @@ def _background_populate_rooms_creator_column_txn(
             for room_id, event_json in room_id_to_create_event_results:
                 event_dict = db_to_json(event_json)
 
+                # The creator property might not exist in newer room versions, but
+                # for those versions the creator column should be properly populate
+                # during room creation.
                 creator = event_dict.get("content").get(EventContentFields.ROOM_CREATOR)
```

Comment:
Aside: Is the `rooms.creator` column useful? I guess it avoids looking up the create event and from the state and event_json tables (though I'd bet we're already looking in those tables anyway).

Reply:
I'm fairly certain that we do use it, but it seems like it has been around forever.

Reply:
Hmm, maybe not -- it was added in #10697.

Reply:
It seems to be mentioned on #10737 too.